knowledge/animals/birds/black_capped_checkadee/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: Hashim Hassany
```

Comment:
Please add your GitHub username here.

Reply:
I have changed this to my GitHub user name.